app/component/CityWeatherStationContentTable.js

```diff
@@ -0,0 +1,408 @@
+    item => item.name === 'ROAD_SURFACE_TEMPERATURE',
+  );
+
+  const { measuredTime } = airTemperature;
+
+  const localName = station.name;
+
+  const openCameraModal = () => {
+    router.push({
+      ...location,
+      state: {
+        ...location.state,
+        moreInfoModalOpen: true,
+        moreInfoModalTitle: (
+          <table>
+            <tbody>
+              <tr>
+                <td style={{ paddingRight: '10px' }}>
+                  <Icon
+                    img="icon-icon_camera-station"
+                    className="camera-icon"
+                  />
+                </td>
+                <td>{localName}</td>
+              </tr>
+            </tbody>
+          </table>
+        ),
+        moreInfoModalContent: (
+          <>
+            <ImageSlider>
+              {cameras.map(item => (
+                <figure className="slide" key={item.presetId}>
+                  <img
+                    className="camera-img"
+                    src={item.imageUrl}
+                    alt={item.presentationName}
+                    onClick={() => {
+                      window.open(item.imageUrl, '_blank');
+                    }}
+                  />
+                </figure>
+              ))}
+            </ImageSlider>
+            <br />
+            <div
+              aria-hidden="true"
+              className="text-button"
+              onClick={() => router.goBack()}
+            >
+              {`< ${intl.formatMessage({
+                id: 'back',
+                defaultMessage: 'Go bakc',
+              })}`}
+            </div>
+          </>
+        ),
+      },
+    });
+  };
+
+  return (
+    <div className="card">
+      <Card className="padding-small">
+        <CardHeader
+          name={intl.formatMessage({
+            id: 'city-weather-station',
+            defaultMessage: 'Weather station',
+          })}
+          description={localName}
+          icon="icon-icon_weather-station"
+          unlinked
+        />
+        <table className="component-list">
+          <tbody>
+            <tr>
+              {airTemperature && (
+                <td>
+                  <table className="sensor-info">
+                    <tbody>
+                      <tr>
+                        <td>
+                          <FormattedMessage
+                            id="air-temperature"
+                            defaultMessage="Air temperature"
+                          >
+                            {(...content) => `${content}`}
+                          </FormattedMessage>
+                        </td>
+                      </tr>
+                      <tr>
+                        <td>{airTemperature.sensorValue}</td>
+                      </tr>
+                      <tr>
+                        <td>{airTemperature.sensorUnit}</td>
+                      </tr>
+                    </tbody>
+                  </table>
+                </td>
```

Comment:
Same thing here, you could probably wrap this in a function component that takes the message id, sensor value and sensor unit as props.